test/framework/ownerreference_helpers.go

```diff
@@ -89,16 +97,17 @@ func AssertOwnerReferences(namespace, kubeconfigPath string, assertFuncs ...map[
 				allErrs = append(allErrs, errors.Wrapf(err, "Unexpected ownerReferences for %s/%s", v.Object.Kind, v.Object.Name))
 			}
 		}
-		errStrings = ""
-		for _, err := range allErrs {
-			errStrings = fmt.Sprintf("%s\n%s", errStrings, err.Error())
+		err = kerrors.NewAggregate(allErrs)
+		if err != nil {
+			aggregateErrs = err.Error()
 		}
-		return kerrors.NewAggregate(allErrs)
-	}).Should(Succeed(), errStrings)
+		return err
```

Comment:
nit: Why not just return `kerrors.NewAggregate(allErrs)` (and drop aggregateErrs)? 

Gomega will print all the errors without us redundantly adding them optional description